app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user&.authenticate params[:session][:password]
```

Comment:
toán tử `&.` có tác dụng gì em?

Reply:
theo em thì &. làm cho mã nguồn trở nên ngắn gọn hơn tránh việc phải lặp lại user.method

Reply:
Em xem lại slide nhé nó có nói về tác dụng của &. đó

Reply:
&. cho phép gọi đến các phương thức của object mà nó nó thể nil, trả về lỗi NoMethodError nếu phương thức không tồn tại